R/helpers.R

```diff
@@ -53,16 +51,21 @@ model_delete_run <- function(run_id, year) {
   )
   s3_objs_dir_path <- gsub(paste0("s3://", bucket, "/"), "", s3_objs_dir_path)
   s3_objs_dir_path <- gsub("//", "/", s3_objs_dir_path)
-  s3_objs_w_run_id <- unlist(purrr::map(
-    s3_objs_dir_path,
-    ~ aws.s3::get_bucket_df(bucket, .x)$Key
-  ))
-
-  # Delete current version of objects
-  purrr::walk(s3_objs_limited, aws.s3::delete_object)
-  purrr::walk(s3_objs_w_run_id, aws.s3::delete_object, bucket = bucket)
+  s3_objs_w_run_id <- s3_objs_dir_path %>%
+    purrr::map(~ aws.s3::get_bucket_df(bucket, .x)$Key) %>%
+    unlist() %>%
+    purrr::map_chr(~ glue::glue("s3://{bucket}/{.x}"))
+
+  return(c(s3_objs_limited, s3_objs_w_run_id))
```

Comment:
It didn't seem to me to be necessary to produce two different S3 path vectors (`s3_objs_limited` and `s3_objs_w_run_id`) with different formats for the path, since the only difference was that `s3_objs_limited` contained the `s3://<bucket_name>/` prefix while `s3_objs_w_run_id` did not. Perhaps I'm missing something, but adding the prefix to `s3_objs_w_run_id` and combining the two vectors made the interface for the new `model_get_s3_artifacts_for_run` simpler and didn't seem to affect the behavior of `model_delete_run`, so I went ahead and combined them.

Reply:
Hmmm, I don't remember why I did that. I'm sure a reason will rear its head eventually...